shotover/src/transforms/kafka/sink_cluster/shotover_node.rs

```diff
@@ -1,9 +1,16 @@
+use crate::tcp::tcp_stream;
 use crate::transforms::kafka::sink_cluster::kafka_node::KafkaAddress;
 use atomic_enum::atomic_enum;
 use kafka_protocol::messages::BrokerId;
 use kafka_protocol::protocol::StrBytes;
+use rand::rngs::StdRng;
+use rand::{Rng, SeedableRng};
+use rand_core::Error;
```

Comment:
Anytime we dont care about the type of the error but just need to bubble something up so that a process can be aborted/restarted and an error logged we should use the anyhow error.
That is because it is very easy to work with and most errors will automatically convert into this error type.
If we needed a concrete error type that we could match on and handle in different ways then we should use the original error type instead.

In this case we should use `anyhow::Error` instead, since the error is only used for logging and triggering a restart of the task. This will allow us to return other error types in the future if we find that we add more logic that can error.

Reply:
Thanks for the tip!